web-client/src/components/CartItem.tsx

```diff
@@ -0,0 +1,58 @@
+import { Button } from '@/components/ui/button';
+import { Card, CardHeader, CardContent, CardFooter } from '@/components/ui/card';
+import { Trash2 } from "lucide-react";
+
+interface CartItemProps {
+    sockId: number;
+    sockVariantId: number;
+    name: string;
+    size: string;
+    price: number;
+    quantity: number;
+    imageUrl: string;
+    onIncrease: () => void;
+    onDecrease: () => void;
+    onRemove: () => void;
+}
+
+export default function CartItem({
+    name,
+    size,
+    price,
+    quantity,
+    imageUrl,
+    onIncrease,
+    onDecrease,
+    onRemove
+}: CartItemProps) {
+    return (
+        <Card className="w-full mx-auto mb-4" >
+            <CardContent className="flex flex-col gap-4 md:flex-row justify-between p-4">
+                <div className="flex gap-4 items-center">
+                    <img src={imageUrl} alt={name} className="w-24 h-24 object-cover rounded-md" />
+                    <div className="text-lg font-semibold">
+                        <h3 className="mb-1">{name}</h3>
+                        <p className="text-sm mb-1 text-gray-400">Size: {size}</p>
+                        <p className="text-base font-bold">${price.toFixed(2)}</p>
+                    </div>
+                </div>
+                <div className="flex gap-4 items-center">
+                    <div className="flex items-center gap-4">
+                        <Button variant="outline" onClick={onDecrease}>
+                            -
```

Comment:
Instead of using a character, use an icon like this:

```ts
<Button
  variant="outline"
  onClick={onIncrease}
  aria-label={`Increase quantity of ${item.name} by 1`}
>
  <Plus size="16" />
</Button>
```

Reply:
Fixed
<img width="552" alt="image" src="https://github.com/user-attachments/assets/f77f9013-9bfc-4ed7-8357-ccf777c82761">